contracts/interfaces/IBond.sol

```diff
@@ -112,15 +112,18 @@ interface IBond {
 
     /**
         @notice The amount of paymentTokens required to fully pay the contract.
-        @return The amount of paymentTokens.
+        @return amountUnpaid The amount of paymentTokens.
```

Comment:
I think the term "amount" is fine for the name of the function as the amountOwed isn't specific, but when we are returning a specific number of tokens, we should use the term "number" https://www.grammar.com/amount_vs._number. I realize I probably wrote the amount, but during the great re-commenting I was trying to be consistent with this usage as well. what do you think?

more specifically - I think the comment should read "the number of paymentTokens", but the variable name is fine.